dask/array/tests/test_fft.py

```diff
@@ -108,3 +112,25 @@ def test_fft_consistent_names():
     assert same_keys(fft(darr, 5), fft(darr, 5))
     assert same_keys(fft(darr2, 5, axis=0), fft(darr2, 5, axis=0))
     assert not same_keys(fft(darr, 5), fft(darr, 13))
+
+
+@pytest.mark.parametrize("funcname", ["rfft", "irfft"])
+def test_wrap_numpy_real(funcname):
+    func = getattr(npfft, funcname)
+    wfunc = fft_wrapper(func, kind=funcname)
+    assert_eq(wfunc(darr), func(nparr))
+    assert_eq(wfunc(darr2, axis=0), func(nparr, axis=0))
+
+
+@pytest.mark.parametrize("modname", ["numpy.fft", "scipy.fftpack"])
+@pytest.mark.parametrize("funcname", ["fft", "ifft", "hfft", "ihfft"])
+def test_wrap_ffts(modname, funcname):
+    fft_mod = pytest.importorskip(modname)
+    try:
+        func = getattr(fft_mod, funcname)
+    except AttributeError:
+        raise SkipTest()
```

Comment:
Use `pytest.skip('fft method missing')  # or a better message` here instead of raising `SkipTest`